vassal-app/src/main/java/VASSAL/build/module/noteswindow/SecretNotesController.java

```diff
@@ -81,11 +81,21 @@
   public static final int COL_NAME = 2;
   public static final int COL_REVEALED = 3;
 
-  // Date formatter to save and restore date/times in the save file
+  private static final String INTERNAL_DATETIME_FORMAT = "M/d/yy h:mm a";
```

Comment:
Why change the date format?

Reply:
This is a fix, it was `m/d/yy h:mm a` before i.e. `minutes/day/year hours:minutes`. Now it is `month/day/year hours:minutes`.

Reply:
How is this going to affect saved games?

Reply:
If they were saved with minutes > 12 then it will probably break them. Otherwise the minutes will become months.

Reply:
The date's just for display purposes, so fixing the format shouldn't break anything. This ought to be tested with a log from before the change, to check that it's still loadable.